tests/queries/0_stateless/03233_dynamic_in_functions.sql

```diff
@@ -0,0 +1,353 @@
+-- Tags: no-fasttest
+
+set allow_experimental_dynamic_type = 1;
```

Comment:
Depending on how long does this test take to execute with heavy builds it might be useful to split the test into smaller chunks. If it is still fast on heavy builds, then no need for it.

Reply:
Agree. Let me fix builds in the CI and we will see how long this test take to execute

Reply:
This test seems to be quite fast in all builds